src/utils/hooks/useFeaturedCategories.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+import { API_BASE_URL } from '../constants';
+import { useLatestAPI } from './useLatestAPI';
+
+export function useFeaturedCategories() {
+  const { ref: apiRef, isLoading: isApiMetadataLoading } = useLatestAPI();
+  const [featuredCategories, setFeaturedCategories] = useState(() => ({
```

Comment:
why the wrapper ()=>, what is the its function?

Reply:
it is necessary to save the data of the request, the banners custom hook also has it.